docs/content/release-process.md

```diff
@@ -68,57 +65,40 @@ $ ./.github/gh-backport.sh <pull-request> <branch-name>
     git pull
 
     # Create a GPG-signed and annotated tag
-    git tag -s -m "Spin v2.0.1" v2.0.1
+    git tag -s -m "Spin v2.0.0" v2.0.0
 
     # Push the tag to the remote corresponding to fermyon/spin (here 'origin')
-    git push origin v2.0.1
+    git push origin v2.0.0
     ```
 
-    Follow the [wrapping up](#wrapping-up) section to finish off the release process.
-
-## Release Candidate
+   This will trigger the [Release](https://github.com/fermyon/spin/actions/workflows/release.yml) workflow which produces and signs release artifacts and uploads them to a GitHub release.
 
-To create a release candidate for a major/minor version of Spin, you will need to do the following:
-
-1. Create the release branch if not already created. With our branch protection rules this is easiest from the Github UI with the [New Branch button here](https://github.com/fermyon/spin/branches). 
-Otherwise, switch to the branch locally.
-
-1. Update the Spin version with `-rc.N` where `N` is the release candidate number (e.g. `2.0.0-pre0` could be `2.0.0-rc.1`).
-   - Bump the version in Spin's `Cargo.toml`
+1. If this is a major/minor release, switch back to `main` and update the `Cargo.toml` version again, this time to e.g. `2.1.0-pre0` if `2.1.0` is the next anticipated release.  _(Patch and release candidates can skip this step.)_
    - Run `make build update-cargo-locks` so that `Cargo.lock` and example/test `Cargo.lock` files are updated
+   - PR this to `main`
 
-   PR these changes to the release branch ensuring that pull request has a base corresponding to the release branch (e.g. `v2.0`).
+## Write release notes
 
-1. Create a new tag with a `v` and then the version used above, e.g. `v2.0.0-rc.1`. Then, push the tag to the `fermyon/spin` origin repo.
+The [release notes template](./release-notes-template.md) can be used as a guide and starting point.
 
-    As an example, via the `git` CLI:
+Once the GitHub release is created, edit the release with these notes.
 
-    ```
-    # Switch to the release branch
-    git checkout v2.0
-    git pull
+> Note: the GitHub release created by the automation pipeline will come pre-populated with title and changelog. Be sure that the changelog uses the correct previous tag/version. If it does not, edit the release to update the previous tag/version and regenerate the changelog. This auto-generated changelog can be added at the end of the release notes.
 
-    # Create a GPG-signed and annotated tag
-    git tag -s -m "Spin v2.0.0-rc.1" v2.0.0-rc.1
+## Notify downstream projects
```

Comment:
Not sure if it is the release engineer's job to announce in Fermyon Discord as well?

Reply:
It could be.  What do you think?

Reply:
I wasn't sure if they would have the necessary access to post in [announcements](https://discord.com/channels/926888690310053918/950022884045226004). I think we could update it accordingly as the project evolves.